nodes/arduino_handler.py

```diff
@@ -210,12 +214,9 @@ def light_intensity_red_callback(msg): # float 0~1
 # TODO: I want to deprecate this with something more feedback loop oriented:
 # See https://github.com/OpenAgInitiative/openag_brain/issues/270 for details
 def water_level_high_callback(msg): # float 1 / 0
-    command = float(msg.data)
-    # if the high water level is > .5, turn the pump on
-    if command > 0.5:
-        actuator_state["pump_5_water_1"] = True
-    else:
-        actuator_state["pump_5_water_1"] = False
+    command = msg.data
+    # if the high water level is >= .5, turn the pump on
+    actuator_state["pump_5_water_1"] = command >= 0.5
```

Comment:
I have a very strong aversion towards the use of `if True then True else False` statements..